pkgs/tools/misc/parallel/default.nix

```diff
@@ -1,21 +1,14 @@
-{ fetchurl, lib, stdenv, perl, makeWrapper, procps, coreutils }:
+{ fetchurl, lib, stdenv, perl, makeWrapper, procps }:
 
 stdenv.mkDerivation rec {
-  name = "parallel-20200922";
+  pname = "parallel";
+  version = "20210122";
 
   src = fetchurl {
-    url = "mirror://gnu/parallel/${name}.tar.bz2";
-    sha256 = "0wj19kwjk0hwm8bk9yfcf3rpr0314lmjy5xxlvvdqnbbc4ml2418";
+    url = "mirror://gnu/parallel/${pname}-${version}.tar.bz2";
+    sha256 = "1wxkqz6ld1bp0ilvc04vhq99qjay1nl6pbk3qzvp3sjavv9vdwdl";
   };
 
-  patches = [
-    ./fix-max-line-length-allowed.diff
-  ];
-
-  postPatch = ''
-    substituteInPlace src/parallel --subst-var-by coreutils ${coreutils}
```

Comment:
I'm skeptical here. I'm willing to bet we still need these fixes?

Reply:
We may need `coreutils` still in the path to find things, but I believe the patch itself is not needed.

Reply:
I'm still pretty new to nixpkgs.  How can I ensure that coreutils is in path, given there's nothing to substitute it in for?

Reply:
propagatedBuildInputs should help. It places runtime-references into a `nix-support` subdirectory so that they are detected by the hash checker.

https://github.com/tomberek/nixpkgs/blob/621bb74bd8b8a8bc0546b4ebdae62db067802104/pkgs/tools/admin/bash-my-aws/default.nix#L26
https://nixos.org/guides/nix-pills/basic-dependencies-and-hooks.html#idm140737319491072

Reply:
Okay, I added `coreutils` to the `propagatedBuildInputs`.

Reply:
This is now done like this:
```
# Force using non-built-in command
$prg ||= ::which("echo");
:qqx("$prg ".${string}x(($len-1-length $prg)/length $string));
```
So propagatedBuildInputs isn't right here. Better would be to update the wrapper to include coreutils in the path. (alongside procps and perl).